after_success.sh

```diff
@@ -2,6 +2,11 @@
 if [[ -n ${DOCKER_TAG} && ${DOCKER_IMAGE} ]]; then
   docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
   docker push $DOCKER_IMAGE:$DOCKER_TAG;
+  #push master as latest
+  if [ "$DOCKER_TAG" == "master" ]; then
+    docker tag $DOCKER_IMAGE:$DOCKER_TAG $IMAGE;
```

Comment:
it is `docker tag SOURCE_IMAGE[:TAG] TARGET_IMAGE[:TAG]`. If TAG is omitted, it becomes `latest`.